recipes/npx/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "npx" %}
+{% set version = "0.0.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 631b9f9688278b5568ddd9bf37e068fc9727c715aa239a739f690db263d028a2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
```

Comment:
Doesn't this need `importlib_metadata`? (We can safely just install it unconditionally.)